shared/js/browser/macos-communication.js

```diff
@@ -89,8 +89,8 @@ const resolveInitialRender = function () {
     const isIsProtectedSet = typeof protections !== 'undefined';
     const isTrackerBlockingDataSet = typeof trackerBlockingData === 'object';
     const isLocaleSet = typeof locale === 'string';
-    const isPhishingSet = typeof phishingStatus === 'boolean';
-    if (!isLocaleSet || !isUpgradedHttpsSet || !isIsProtectedSet || !isTrackerBlockingDataSet || !isPhishingSet) {
+    const isMaliciousSiteSet = maliciousSiteStatus && maliciousSiteStatus.kind !== undefined;
+    if (!isLocaleSet || !isUpgradedHttpsSet || !isIsProtectedSet || !isTrackerBlockingDataSet || !isMaliciousSiteSet) {
         return;
```

Comment:
This looks like a breaking change, will it also affect iOS (since it uses the same base comms).

Reply:
It does affect iOS, but the native-side change is cross-platform on BSK, which on iOS will return `maliciousSiteStatus.kind: null`.

Reply:
I’ve added a check just in case, till iOS is ready for release: https://github.com/duckduckgo/privacy-dashboard/pull/250/commits/1f7aef071e7bd44db82b83a787b5682a3397ded8